desktop/appconfig.cpp

```diff
@@ -0,0 +1,55 @@
+#include "appconfig.h"
+
+#include <QLoggingCategory>
+
+
+const QStringList loggingCategories = 
+  {"UIManager", 
+    "Flexbox", 
+    "WebSocketModule", 
+    "Networking", 
+    "ViewManager", 
+    "RCTNotification", 
+    "default",
+    "RCTStatus",
+    "jsserver",
+    "status"};
+Q_LOGGING_CATEGORY(APPCONFIG, "AppConfig")
+
+AppConfig* AppConfig::appConfig = nullptr;
+
+AppConfig::AppConfig()
+  : loggingEnabled(false) {
+  
+}
+
+AppConfig* AppConfig::inst() {
+  if (appConfig == nullptr) {
```

Comment:
If we use following:
```
static AppConfig appConfig;
return &appConfig
```
Then we can remove the `static AppConfig* appConfig;` below.

Reply:
Thanks! Replaced instance pointer with a reference.